### PR TITLE
Add icons page to docs

### PR DESCRIPTION
**Pull request by jcfilben**
Documents props and theme props that can be used with grommet icons

related to https://github.com/grommet/grommet-icons/issues/305

### CODE REVIEW

File: src/screens/Docs/Icons.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        <Property name="height">
+          <Description>
+            The amount of spacing between icon and label in the anchor.
+          </Description>
+          <PropertyValue type="string">
+            <Example>"24px"</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="width">
```
Comment (taysea) [inline src/screens/Docs/Icons.js:63]: For now, Eric and I discussed not documenting `width`.

File: src/screens/Docs/Icons.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            <Example defaultValue>"medium"</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="height">
+          <Description>
+            The amount of spacing between icon and label in the anchor.
+          </Description>
+          <PropertyValue type="string">
+            <Example>"24px"</Example>
```
Comment (taysea) [inline src/screens/Docs/Icons.js:59]:
```suggestion
            <Example>"medium"</Example>
```

File: src/screens/Docs/Icons.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        <Property name="size">
+          <Description>Size of the icon.</Description>
+          <PropertyValue type="string">
+            <Example defaultValue>"medium"</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="height">
+          <Description>
+            The amount of spacing between icon and label in the anchor.
```
Comment (taysea) [inline src/screens/Docs/Icons.js:56]:
```suggestion
            A text t-shirt size to align the height of an icon with the text line-height. This is useful when an icon's dimensions is smaller than the text line-height and the icon is presented in a row with align="start".
```

File: src/screens/Docs/Icons.js (hunk 2 above)
Comment (taysea) [reply src/screens/Docs/Icons.js:59]: `px` values aren't supported right now, only t-shirt sizes

File: src/screens/Docs/Icons.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+      <ThemeDoc>
+        <Property name="icon.extend">
+          <Description>Any additional styles to place on the icon.</Description>
+          <GenericExtend />
+        </Property>
+
+        <Property name="icon.size">
+          <Description>
+            The text color used to style active elements.
```
Comment (taysea) [inline src/screens/Docs/Icons.js:81]:
```suggestion
            The height and width of the icon.
```

File: src/screens/Docs/Icons.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+    medium: string;
+    large: string;
+    xlarge: string;
+}`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="icon.disableScaleDown">
+          <Description>
+            When set to true `vector-effect="non-scaling-stroke"` is added to
```
Comment (taysea) [inline src/screens/Docs/Icons.js:95]:
```suggestion
            When set to true, `vector-effect="non-scaling-stroke"` is added to
```

File: src/screens/Docs/Icons.js [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+            simple icons. Currently this affects Add, Checkmark, Close, Down,
+            LinkDown, LinkNext, LinkUp, Next, Previous, Search, and Up.
+          </Description>
+          <GenericBoolFalse />
+        </Property>
+
+        <Property name="icon.matchSize">
+          <Description>
+            When true Grommet components that have a size prop and an icon will
```
Comment (taysea) [inline src/screens/Docs/Icons.js:104]:
```suggestion
            When true, Grommet components that have a size prop and an icon will
```

**Comment by taysea** [comment]
Clicking on `Icons` in search navigates to grommet-icons site as opposed to the icons docs page.

File: src/screens/Docs/Icons.js (hunk 4 above)
Comment (jcfilben) [reply src/screens/Docs/Icons.js]: Fixed

**Review by taysea — APPROVED** [review]
Thanks for adding this. Looks good.

**Review by ericsoderberghp — APPROVED** [review]

**Pull request merged**